apix-interface/src/main/java/eu/xenit/apix/comments/Conversation.java

```diff
@@ -0,0 +1,70 @@
+package eu.xenit.apix.comments;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Conversation {
+
+    private List<Comment> comments;
+    private boolean hasMore;
+    private boolean canCreate;
+
+    public List<Comment> getComments() {
+        return comments;
+    }
+
+    public void setComments(List<Comment> comments) {
+        this.comments = comments;
+    }
+
+    public Conversation comments(List<Comment> comments) {
```

Comment:
If you want to use a builder pattern, I would have expected a more explicit implementation of that, with a separate `ConversationBuilder` class.

Reply:
This is more out of a desire for something like properties rather than an actual builder (that seems overkill). Ill remove them to keep the code clean